vector/src/main/res/xml/vector_settings_security_privacy.xml

```diff
@@ -99,7 +102,8 @@
         <im.vector.app.core.preference.VectorPreference
             android:key="SETTINGS_ENCRYPTION_IMPORT_E2E_ROOM_KEYS_PREFERENCE_KEY"
             android:summary="@string/encryption_import_room_keys_summary"
-            android:title="@string/encryption_import_e2e_room_keys" />
+            android:title="@string/encryption_import_e2e_room_keys"
+            app:isPreferenceVisible="false" />
```

Comment:
I think this line is an error, please revert it. We keep displaying the import e2e room keys option